app/docker/views/containers/create/createContainerController.js

```diff
@@ -544,14 +548,9 @@ function ($q, $scope, $state, $timeout, $transition$, $filter, Container, Contai
 
     SystemService.info()
     .then(function success(data) {
-      var runtimes = data.Runtimes;
-      $scope.availableRuntimes = runtimes;
-      if ('runc' in runtimes) {
-        $scope.config.HostConfig.Runtime = 'runc';
-      }
-      else if (Object.keys(runtimes).length !== 0) {
-        $scope.config.HostConfig.Runtime = Object.keys(runtimes)[0];
-      }
+      $scope.availableRuntimes = Object.keys(data.Runtimes);
+      $scope.availableRuntimes.unshift('Default');
+      $scope.config.HostConfig.Runtime = 'Default';
```

Comment:
See my comment below about simply setting the `$scope.config.HostConfig.Runtime` to `''`.